Doc: Environment variables cannot be in config.string comments. (#16689)
elastic/logstash: docs/static/pipeline-configuration.asciidoc

  # ...
}
----------------------------------

> NOTE: Comments containing environment `${var}` references in `config.string` are not allowed. Remove the `$` sign to avoid pipeline loading failures.

Reviewer: Same as above:

```suggestion
NOTE: Comments containing environment `${var}` references in `config.string` will still be evaluated. Remove the `$` sign to avoid pipeline initialization failures.
```

> NOTE: Comments containing environment `${var}` references in `config.string` are not allowed. Remove the `$` sign to avoid pipeline loading failures.

Author: Combined both of your suggestions into one. WDYT?
```suggestion
NOTE: Comments containing environment variable `${var}` references in `config.string` will still be evaluated.
Remove the `$` sign to avoid pipeline loading failures.
```

  # ...
}
----------------------------------

> NOTE: Comments containing environment variable `${var}` references in `config.string` will still be evaluated.

Reviewer 2: ```suggestion
NOTE: Comments containing environment variable `${var}` references in `config.string` are still evaluated.
```
